ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -127,6 +159,21 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service
+- as: aws-ipi-disc-priv-tp-f28
```

Comment:
it is a TP job, which does not support major version upgrade.

Reply:
Missed it. thanks